d2l-icon.html

```diff
@@ -105,7 +111,7 @@
 				this.async(function() {
 					this.__initialize();
 					this.updateStyles(); // need to re-calc iron-icon variables
-				}, 1);
```

Comment:
I found that this delay is necessary in some cases when the icon property changes. I have had the RTL not working in d2l-table and d2l-icon-button because of this